src/Utils/SchemaExtender.php

```diff
@@ -203,6 +198,7 @@ protected static function extendInputFieldMap(InputObjectType $type): array
 
         $extensions = static::$typeExtensionsMap[$type->name] ?? null;
         if ($extensions !== null) {
+            /** @var InputObjectTypeExtensionNode $extension */
```

Comment:
maybe we could use `assert()`s instead of `@var`

Reply:
I added comments to explain why this must hold true.